docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -92,6 +92,7 @@ Now let's make a new project using our versioning system. Just type:
 
     mkdir myproject.git
     git init --bare myproject.git
+    mkdir myproject
```

Comment:
The directories myproject & myproject.git are at the same level.

On 9/6/2016 7:21 AM, Wasabi Fan wrote:
>
> In docs/tutorials/setting-up-python-pycharm.md 
> <https://github.com/ev3dev/ev3dev.github.io/pull/233#discussion_r77632353>:
>
> > @@ -92,6 +92,7 @@ Now let's make a new project using our versioning system. Just type:
> >
> >      mkdir myproject.git
> >      git init --bare myproject.git
> > +    mkdir myproject
>
> Is the idea that the |myproject.git| directory is inside |myproject|? 
> If so, this line should go at the beginning and there should be a |cd 
> myproject| line after it. Or am I misunderstanding?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ev3dev/ev3dev.github.io/pull/233/files/13b719c211ff59f300fd3ab804c7a57250d528b0#r77632353>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AAxESrums96UNwPW4gTgx1cnQuRJV7VXks5qnWjUgaJpZM4J1dC2>.
>